odoo/addons/base/views/ir_module_views.xml

```diff
@@ -162,21 +160,19 @@
                   <templates>
                     <t t-name="kanban-menu">
                         <t t-set="installed" t-value="record.state.raw_value == 'installed'"/>
-                        <a type="edit" class="dropdown-item">Module Info</a>
+                        <a type="open" class="dropdown-item">Module Info</a>
                         <a t-if="record.website.raw_value" role="menuitem" class="dropdown-item o-hidden-ios" t-att-href="record.website.raw_value" target="_blank">Learn More</a>
                         <a t-if="installed" name="button_immediate_upgrade" type="object" role="menuitem" class="dropdown-item" groups="base.group_system">Upgrade</a>
                         <a t-if="installed" name="button_uninstall_wizard" type="object" role="menuitem" class="dropdown-item" groups="base.group_system">Uninstall</a>
                     </t>
-                    <t t-name="kanban-box">
+                    <t t-name="kanban-card">
                       <div class="oe_module_vignette">
                         <div class="oe_module_icon">
```

Comment:
try to remove those classnames (use bootstrap utility classes instead), and remove the associated css